packages/shopify-api/lib/billing/subscriptions.ts

```diff
@@ -9,13 +9,51 @@ import {
 } from './types';
 
 const SUBSCRIPTION_QUERY = `
-  query appSubscription {
-    currentAppInstallation {
-      activeSubscriptions {
+query appSubscription {
+  currentAppInstallation {
+    activeSubscriptions {
+      id
+      name
+      test
+      lineItems {
         id
-        name
-        test
+        plan {
+          pricingDetails {
+            ... on AppRecurringPricing {
+              discount {
+                durationLimitInIntervals
+                remainingDurationInIntervals
+                priceAfterDiscount {
+                  amount
+                }
+                value {
+                  ... on AppSubscriptionDiscountAmount {
+                    amount {
+                      amount
+                      currencyCode
+                    }
+                  }
+                  ... on AppSubscriptionDiscountPercentage {
+                    percentage
+                  }
+                }
+              }
```

Comment:
the type https://github.com/Shopify/shopify-api-js/pull/1234/files#diff-e366dcff14ef93ab7766ac21e2c0bf9c179386a6728a7dc92b54597e57f4dc3cR340-R353

has `price` and `interval`, do we need to include those fields here?